app/models/event.rb

```diff
@@ -132,6 +132,10 @@ def use_a_template
     end
   end
 
+  def timecards
```

Comment:
I would prefer to have this in the Timecard model as a scope, where it can be reused.
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/models/availability.rb#L14
Also, I worry that people will see `@event.timecards` and think that it is a `has_many/belongs_to` relationship, which it is not.
Finally, you'll need to filter out cancelled timecards.

Reply:
@kgf I was thinking about a scope initially, I will attempt this again using that methodology and fix the logic as well.